microsoft/atlas-design
# [bug] Change spacing atomics output order

Task: task-[428733](https://dev.azure.com/ceapex/Engineering/_boards/board/t/Fox%20Team/Stories/?workitem=428733)

Link: preview-[199](https://design.docs.microsoft.com/pulls/199)

This PR changes the order to ensure the correct specificity. `padding-none` was overriding classes like `padding-xl-tablet` at all screen widths.

## Testing

1. Visit the preview link above. Inspect an element with DevTools and add the following classes: `padding-none padding-top-s padding-left-m-tablet padding-right-xl-desktop`. The same should work with `margin`.
2. Change the screen width and verify that there's `top` padding in mobile, `top and left` padding in tablet, `top, left, and right` padding in desktop.

![image](https://user-images.githubusercontent.com/64044661/119032584-33fb3700-b961-11eb-8742-ec46d307bffd.png)
